File: src/router/routes/validuser.go
package routes

import (
	"router/sqldb"
)

//IsValidUser returns whether user is valid or not
func IsValidUser(email string,token string) bool{
	if(!VerifyToken(token)){
		return false;
	}

	stmt := "select * from sureify.user where email = '"+email+"' and token='"+token+"'";
	rows, _ := sqldb.DB.Query(stmt);
	userExists := rows.Next()
	return userExists;
}

File: api/config/validuser.php
<?php

    require("../index.php");

    function isValidUser($conn,$token){

        $email = verifyToken($token);
        
        $sql = "select * from user where email='$email' and token='$token'";
        $res = $conn->query($sql);

        if($res->num_rows>0){
            return $email;
        }else{
            return null;
        }
    }
?>